docs/DeveloperGuide.md

```diff
@@ -1,29 +1,165 @@
 # Developer Guide
 
+* [Acknowledgements](#acknowledgements)
+* [Setting up, getting started] (#setting-up-getting-started)
+* [Design & implementation](#design--implementation)
+  * [Architecture](#architecture)
+  * [Category feature](#category-feature)
+* [Product scope](#product-scope)
+  * [Target user profile](#target-user-profile)
+  * [Value proposition](#value-proposition)
+* [User Stories](#user-stories)
+* [Non-Functional Requirements](#non-functional-requirements)
+* [Glossary](#glossary)
+* [Instructions for manual testing](#instructions-for-manual-testing)
+
 ## Acknowledgements
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
+## Setting up, getting started
+
+Refer to the guide [Setting up and getting started]().
+
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Architecture
+
+The UML diagram below shows the main relationships between the classes in the Brokeculator application.
+![img.png](images/architecture.png)
+
+### Category feature
+**Implementation** </br>
```

Comment:
The `</br>` tag is not rendered properly. It shows as text in the webpage. Perhaps consider puttting this tag on the next line?
![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/110613918/7c96fd83-e129-44d0-bfdf-82a77fbd48f6)